src/auth/ducks/selectors.ts

```diff
@@ -63,3 +63,12 @@ export const getUserEmail = (
   }
```

Comment:
Is the username already passed through this route from the backend?

Reply:
💭 If we implemented contract testing for SFTT we would know this through testing, it might be worthwhile implementing it some day

Reply:
Yeah, https://github.com/Code-4-Community/speak-for-the-trees-backend-v2/pull/52